llpc/lower/llpcSpirvLowerRayTracing.cpp

```diff
@@ -642,36 +650,42 @@ bool SpirvLowerRayTracing::runImpl(Module &module) {
 }
 
 // =====================================================================================================================
-// Create global variable for the TraceParam
-void SpirvLowerRayTracing::createGlobalTraceParams() {
+// Create alloc variable for the TraceParam
+void SpirvLowerRayTracing::createTraceParams(Function *entryFunc) {
+  Instruction *insertPos = &*(entryFunc->begin()->getFirstInsertionPt());
+  m_builder->SetInsertPoint(insertPos);
```

Comment:
Use m_builder->SetInsertPointPastAllocas

Reply:
CreateAlloca() can be placed at the start of function

Reply:
I suppose it's fine in this case because it's really just creating `alloca`s, not code to initialize their contents.